JapaneseFilter/sections/specific.txt

```diff
@@ -31,6 +31,10 @@ news24.jp,michill.jp,fnn.jp,bengo4.com,zigsow.jp,navitime.com,cazual.shufu.co.jp
 !
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
+||top.bcdn.jp/i/saiyou/200424_saiyou.png
+biglobe.ne.jp###join-bnr
+biglobe.ne.jp###spage-r
```

Comment:
I think we should not block this element

Reply:
Maybe okay for Annoyances, but I'd block `###spage-bnr` than `###spage-r` as the former is more annoying (these two are the same).

Reply:
Please move to annoyances.txt with a comment